packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart

```diff
@@ -266,14 +268,22 @@ abstract class PlatformLocation {
   /// The base href where the Flutter app is being served.
   ///
   /// See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base
-  String getBaseHref();
+  String? getBaseHref();
```

Comment:
Checking if getBaseHref can be null

Reply:
It can:

https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/navigation/url_strategy.dart#L246